WikiWatershed/model-my-watershed
# Drop some AOI selections for BigCZ

## Overview

This PR removes some AOI selection possibilities from BigCZ, leaving them as is in MMW. Per #2400, we keep the USGS HUCs as well as county lines but drop school districts and congressional districts. We also drop DRB RWD, leaving the NHD version.

Connects #2400 

### Demo

![drb-draw-tools](https://user-images.githubusercontent.com/4165523/31900658-2f34e442-b7ed-11e7-812e-c9fde7538863.gif)

### Notes

It felt nicer to do the known boundary filter using an ES5 filter rather than underscore's, so I dropped the underscore import from there, too.

## Testing Instructions
- get this branch, then bundle
- visit mmw and verify that all the draw tools are still in place
- visit bigcz and verify that the selections now match the changes requested in #2400
